### PR TITLE
Modify use local socket.io.js file

### PR DESCRIPTION
**Pull request by tooto1985**
If not use default 80 port, modify use local socket.io.js file.

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
     <script src="//cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/jqueryui-touch-punch/0.2.3/jquery.ui.touch-punch.min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/howler/2.0.0/howler.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/socket.io/1.4.8/socket.io.min.js"></script>
+    <script src="/socket.io/socket.io.js"></script>
```
Comment (debugger22) [inline app/index.html]: Where is this file?

Comment (tooto1985) [inline app/index.html]: http://stackoverflow.com/questions/17757728/where-is-the-socket-io-client-side-js-file-located

Comment (tooto1985) [inline app/index.html]: File is in the "socket.io-client" module's  socket.io.js.

Comment (debugger22) [inline app/index.html]: Oh this is good but unfortunately we should rather prefer CDN.

**Pull request closed without merging**